.woodpecker/docs.yml

```diff
@@ -1,3 +1,5 @@
+version: 1
+
```

Comment:
```suggestion
```

have some with none set to test agains

Reply:
Well yes our tests should cover all usecases ... and we must test against what most users do&will use